CoffeeMachineStates/src/States.java

```diff
@@ -0,0 +1,9 @@
+public enum States {
+    Off,
```

Comment:
@AraChilingaryan jan good practice is to have enum variables uppare case for example "OFF" or "SELECT_COFFEE"